Ocarina/OcarinaManager.swift

```diff
@@ -35,6 +35,8 @@ public class OcarinaManager: NSObject {
         }
     }
     
+    public var userAgent = ""
```

Comment:
Could you make user agent an optional instead? It makes the code a lot nicer :)

```suggestion
    public var userAgent: String?
```

Reply:
All please add some documentation on what the property is for. When using Xcode you should be able to do that by putting the cursor on line 38 and pressing CMD+?. This will insert the documentation comment.